src/molecules/tables/filter-column.tsx

```diff
@@ -152,14 +213,7 @@ export default function FilterColumn({
       >
         {column.value !== '' ? (
           <div className="border px-3 py-1 border-gray-300 rounded-full text-xs mr-2 flex justify-center">
-            <DropdownMenuTrigger>
-              <span className="font-semibold">{column.displayName}</span>:{' '}
-              {column.type === 'date'
-                ? column.value
-                  ? new Date(column.value).toLocaleDateString()
-                  : ''
-                : column.value}
-            </DropdownMenuTrigger>
+            <DropdownMenuTrigger>{dropDownCTA(column)}</DropdownMenuTrigger>
```

Comment:
<DropdownCta />


This provides more performance because this way react will know that this is a react component. Also it let you use react life cycle methods.

Reply:
Thanks for the comment! 👍